patterns/1-initial/migrating-from-innersource-to-open-source.md

```diff
@@ -0,0 +1,125 @@
+## Title
+
+Migrating from InnerSource to Open Source
+
+## Patlet
+
+When an InnerSource project succeeds internally and meets criteria for external release, establish a process that addresses legal, security, governance, and community readiness to transition the project to open source while maintaining its internal value.
```

Comment:
If you were to split this into 2 sentences, how would that look like?

1st sentence / problem
2nd sentence / solution